.buildkite/release-pipelines/start-code-freeze.yml

```
@@ -0,0 +1,17 @@
+steps:
+  - label: Start Code Freeze
+    plugins:
+      - $CI_TOOLKIT_PLUGIN
+    # The first client to implement releases in CI was Android so the automation works in that queue.
+    # We might want to move it to a leaner one in the future.
+    agents:
+        queue: android
+    command: |
+      echo '--- :git: Configure Git for release management'
+      .buildkite/commands/configure-git-for-release-management.sh
+
+      echo '--- :ruby: Setup Ruby tools'
+      install_gems
+
+      echo '--- :shipit: Start code freeze'
+      bundle exec fastlane code_freeze skip_confirm:true
```

We recently started disallowing retries from Buildkite ([example from DOApple](https://github.com/bloom/DayOne-Apple/pull/24201)) so that the release driver manages the build from Releases V2 instead: 

(I _think_ the indenting should work as expected, but with yaml, 🤷‍♂️)

```suggestion
      bundle exec fastlane code_freeze skip_confirm:true
    retry:
      manual:
        # If those jobs fail, one should always prefer re-triggering a new build from ReleaseV2 rather than retrying the individual job from Buildkite
        allowed: false
```

---

3ad44174